controller/workitem.go

```diff
@@ -633,6 +634,14 @@ func ConvertWorkItemsToCSV(ctx context.Context, app application.Application, all
 	}
 	columnLabels = sortedLabels
 	columnKeys = sortedKeys
+	// add the childs numbers manually as the third column
+	const childsNumbersKey = "_childs"
```

Comment:
Let's use the correct plural form: children

Reply:
Fixed.